tests/models/test_detection.py

```diff
@@ -34,3 +38,122 @@ def test_fasterrcnn_bbone_train(tmpdir):
 
     trainer = pl.Trainer(fast_dev_run=True, default_root_dir=tmpdir)
     trainer.fit(model, train_dl, valid_dl)
+
+
+def _create_yolo_config_file(config_path):
+    config_file = open(config_path, 'w')
+    config_file.write(
+        '''[net]
```

Comment:
can we rather have it in `tests/data` no need to be as a code

Reply:
That's a good idea. I moved the test config to `tests/data`.